templates/web/pages/product_edit/product_edit_form_display.tt.html

```diff
@@ -207,7 +207,11 @@ <h1>[% title %]</h1>
                                     <td>
                                         <!--label starts-->
                                         [% IF nutriment.name.defined %]
-                                            <label class="nutriment_label" for="nutriment_[% nutriment.enid %]">[% nutriment.prefix %][% nutriment.name %]</label>
+                                        [% IF nutriment.nid == 'fat' || nutriment.nid == 'saturated-fat' || nutriment.nid == 'salt'|| nutriment.nid =='sugars' %]
+                                            <label class="nutriment_label" for="nutriment_[% nutriment.enid %]">[% nutriment.prefix %][% nutriment.name %]*</label>
+                                        [% ELSE %]
+                                        <label class="nutriment_label" for="nutriment_[% nutriment.enid %]">[% nutriment.prefix %][% nutriment.name %]</label>
+                                        [% END %]  
```

Comment:
```suggestion
                                        [% ELSE %]
                                            <label class="nutriment_label" for="nutriment_[% nutriment.enid %]">[% nutriment.prefix %][% nutriment.name %]</label>
                                        [% END %] 
```

cc @yuktea @stephanegigandet